### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by fpandza** [taotaonice/FaceShifter#23]
Hi Taotanice,

Firstly thank you for providing the pretrained generator model.

Secondly, I've tried running the demo_inference.py and the results are pretty inconsistent and not that great.
I also tried adjusting and running the code from online_preview.py, which has a bit more going on with the mask and everything, but without much success. I did find that opening target image in BGR mode made the switch more aggresive but resulted in more artifacts.

I'm wondering what is your primary mode of testing the performance of the model?

**Comment by taotaonice** [comment]
human eyes, on many test cases
But I know it is not rigorous. 233333333

**Comment by ZXXYy** [comment]
Hi all,
where can I find the pretained generator model? Thank you so much. 
Could you please send me a pretained model?

**Comment by taotaonice** [comment]
leave your email

**Comment by HelloJianHan** [comment]
Could you please send me a pretained model?
My email is jianhan.thu@gmail.com

**Comment by Daic115** [comment]
> leave your email
d214939381@gmail.com,  thanks！

**Comment by dragen1860** [comment]
@taotaonice liangqu.long@gmail.com thank you very much.

**Comment by CHUUUU** [comment]
@taotaonice 

lmj5929naver@gmail.com 

thank you !!

**Comment by lo2aayy** [comment]
@taotaonice 
lo2aayyguc@gmail.com

Thanks!

**Comment by mkaskov** [comment]
@taotaonice
Could you please send me a pretrained model?
max8mob@gmail.com

Thank you.

**Comment by jiusigua** [comment]
@taotaonice
Hi, I‘m studying FaceShifter now. Could you please send me a pretrained model?
my email is: 1515774621@qq.com.   Thanks !
Looking forward to your reply !!!

**Comment by PayloadJF** [comment]
@taotaonice 
Could you please send me a pretrained model?
jifan1994@gmail.com
Thank you very much.

**Comment by lzwhard** [comment]
> leave your email

lzwhard@163.com

Tks a lot!

**Comment by SCHRODlNGER** [comment]
samanyupobbati@gmail.com
Thanks a lot!!

**Comment by TriLoo** [comment]
songmhor@gmail.com

Thank you very much.

**Comment by PVSemk** [comment]
pvsemk@mail.ru
Thanks in advance!

**Comment by songyoyo** [comment]
Could you please send me a pretrained model?
610240943@qq.com

Thank you very much!

**Comment by nevidimov** [comment]
@taotaonice
Please share a pretrained model to nevitas0@gmail.com. If possible, both generator model and HEAR dataset. Thank you so much!

**Comment by luoqiuluoqiu** [comment]
Could you please send me a pretrained model?
871602547@qq.com

Thank you very much!

**Comment by clumsynope** [comment]
@taotaonice
Could you please share your pretrained model to clumsynope@126.com. Thank you!

**Comment by mugil123** [comment]
Hi @taotaonice 
Could you please share the pretrained model to mugil123@yahoo.com.
Appreciate it
Thanks

**Comment by koka0630** [comment]
Could you please share the pretrained model to youmokakan.220.284@gmail.com?
Thank you so much!

**Comment by Bo396543018** [comment]
Can anyone share your pretrained model to chenbo940902@gmail.com？
3Q

**Comment by ayush-angelium** [comment]
@taotaonice Can you please share the pretrained model to asharma@noborders.net ?

**Comment by liangying1026** [comment]
Can anyone share the pretrained model to ten26angel@gmail.com？
Thanks

**Comment by ganyibo** [comment]
Can anyone share the pretrained model to gyb15079896285@163.com？
Thanks.

**Comment by tandohero** [comment]
Could you please send the trained weights to tandohero@gmail.com?

I can upload to google drive/dropbox if you like and share here?

**Comment by NQWHR** [comment]
@taotaonice Can you please share the pretrained model to 1127079185@qq.com? Thank you very much!

**Comment by sumanchoudhurynitr** [comment]
> leave your email

@taotaonice : Could you please send the pre-trained model to sumanchoudhury.nitr@gmail.com ? Thank you !

**Comment by shiluooulihs** [comment]
@taotaonice Could you please send the pre-trained model to 1127901623@qq.com ? Thank you !

**Comment by yangyanglisjtu** [comment]
@taotaonice Could you please send the pre-trained model to yangyanglisjtu@outlook.com ? Thank you !

**Comment by AkiraTOSEI** [comment]
@taotaonice  Hi! Thank you for reproducing the code. I learned a lot. Could you please send the pre-trained model to yosouguy0926@yahoo.co.jp? Thank you !

**Comment by Mohamed209** [comment]
thanks for the demo , here is my mail , would you please send me the weights to test the model
mohamedmosad209@gmail.com

**Comment by AlexeiKaplenko** [comment]
@taotaonice
Could you please send me a pretrained model?
xkaple00@gmail.com
Thank you very much.

**Comment by buzhangjiuzhou** [comment]
@taotaonice
Could you please send me a pretrained model and some detailed description of it?
972161836@qq.com

**Comment by sywh** [comment]
@taotaonice 
Could you please share your pretrained model with me? Thanks a lot!
zl_wol@163.com

**Comment by JohnGates1985** [comment]
@taotaonice Could you please send the pre-trained model to hohn.gates1985@gmail.com ? Thank you !

**Comment by xiangliu886** [comment]
@taotaonice Could you please send the pre-trained model to 648960862@qq.com? Thank you !

**Comment by coranholmes** [comment]
@taotaonice Thank you. Could you please send me the pre-trained model too? My email is coran.holmes#gmail.com (please replace # with @)

**Comment by eyetye** [comment]
Could you please send me a pre-trained model?
My email is 1439292242@qq.com
Thanks a lot!

**Comment by meet-cjli** [comment]
@taotaonice  Could you please send me a pretrained model? my email is: 397923342@qq.com. Thanks !

**Comment by xuguozhi** [comment]
575760679@qq.com
好人一生平安
A good man will be a lucky man during his whole life

**Comment by wgh1989** [comment]
w_gh1989@126.com
thanks very much!

**Comment by wjl520** [comment]
@taotaonice Hi! Thank you for reproducing the code. I learned a lot. Could you please send the pre-trained model to 2064166884@qq.com? Thank you !

**Comment by SajjadAemmi** [comment]
Can anyone share the pretrained model to 
sajjadaemmi@gmail.com ？
Thanks.

**Comment by wjl520** [comment]
Can anyone share the pretrained model to
jielong688@gmail.com ？
Thanks.

**Comment by arkouda** [comment]
Hey @taotaonice,

Can you share the pretrained model to burnwat3r@gmail.com?
Thanks.

**Comment by alwan1989** [comment]
@taotaonice
Please share a pretrained model to 185919527@qq.com. If possible, both generator model and HEAR dataset. Thank you so much!

**Comment by ltskzy** [comment]
Can you share the pretrained model to 617786386@qq.com?
Thanks very much.

**Comment by kjaleph** [comment]
Please can anybody share pretrained model to cu.17bcs1647@gmail.com
@coranholmes @dragen1860 @eyetye @nevidimov @xiangliu886 @taotaonice

**Comment by kjaleph** [comment]
> Could you please send me a pretained model?
> My email is [jianhan.thu@gmail.com](mailto:jianhan.thu@gmail.com)

 Please send me model weights also I want to try it out. cu.17bcs1647@gmail.com thanks !

**Comment by PlatinumYzm** [comment]
@taotaonice Could you share the pretrained model weights to smile840761772@gmail.com? Thank you very much!

**Comment by alwan1989** [comment]
Can you share the pretrained model to 185919527@qq.com?
Thanks.

**Comment by wangrui5781** [comment]
@taotaonice  Could you please send me a pretrained model?
605879459@qq.com
Thank you.

**Comment by payne4handsome** [comment]
@taotaonice
 Share your pretrained model to sunny4handsome@163.com too. Thanks very much!

**Comment by qdushuaicao** [comment]
Please can anybody share pretrained model to caoshuai1994@yeah.net .Thanks very much!@coranholmes @dragen1860 @eyetye @nevidimov @xiangliu886 @taotaonice

**Comment by qqwya** [comment]
Can anyone share the pretrained model to
786256773@qq.com ？
Thanks.

**Comment by V1len** [comment]
@taotaonice
Could you please send me a pretrained model?
1667150169@qq.com
Thank you very much.

**Comment by Penkey-1223** [comment]
Can anyone  send me a pretrained model? 
I need it very much. 
@taotaonice 
@coranholmes @dragen1860 @eyetye @nevidimov @xiangliu886 
Thanks a lot!
1148872693@qq.com     or
lesterlsd@outlook.com

**Comment by MasterXiYu** [comment]
Could you please send me a pretained model?
My email is 1532940772@qq.com

**Comment by Penkey-1223** [comment]
I am sorry to tell you that I haven't received it yet @MasterXiYu

**Comment by MasterXiYu** [comment]
ok......




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年5月24日(星期一) 凌晨0:14
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [taotaonice/FaceShifter] Demo (#23)





 
I am sorry to tell you that I haven't received it yet @MasterXiYu

**Comment by Penkey-1223** [comment]
If you get it, please share it with me. Thanks!@MasterXiYu

**Comment by MasterXiYu** [comment]
got it,thanks



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年5月24日(星期一) 凌晨0:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [taotaonice/FaceShifter] Demo (#23)





 
If you get it, please share it with me. Thanks!@MasterXiYu

**Comment by melody-rain** [comment]
@taotaonice nice work!
Could you please send me a pretrained model?
xiah_sunny@hotmail.com
Thank you very much.

**Comment by alcanunsal** [comment]
@taotaonice could you send me the pretained models? alcanunsal@gmail.com Thanks in advance!

**Comment by MasterXiYu** [comment]
您好，易旭已经收到您的邮件，谢谢。This is an automatic reply, confirming that your e-mail was received.Thank you
